matchmake-extension/auto_matchmake_postpone.go

```diff
@@ -0,0 +1,102 @@
+package matchmake_extension
+
+import (
+	"math"
+
+	"github.com/PretendoNetwork/mario-kart-7-secure/globals"
+	nex "github.com/PretendoNetwork/nex-go"
+	match_making "github.com/PretendoNetwork/nex-protocols-go/match-making"
+	matchmake_extension "github.com/PretendoNetwork/nex-protocols-go/matchmake-extension"
+	"github.com/PretendoNetwork/nex-protocols-go/notifications"
+	common_globals "github.com/PretendoNetwork/nex-protocols-common-go/globals"
+	"encoding/json"
+)
+
+func AutoMatchmake_Postpone(err error, client *nex.Client, callID uint32, matchmakeSession *match_making.MatchmakeSession, message string) {
+	missingHandler := false
+	if commonMatchmakeExtensionProtocol.CleanupSearchMatchmakeSessionHandler == nil {
+		logger.Warning("MatchmakeExtension::AutoMatchmake_Postpone missing CleanupSearchMatchmakeSessionHandler!")
+		missingHandler = true
+	}
+	if missingHandler {
+		return
+	}
```

Comment:
I see the benefit in storing `missingHandler` for when checking for multiple handlers, but in cases where there's only one check maybe we should just bail right away and not create any unnecessary variables?